composer.json

```diff
@@ -12,8 +12,10 @@
         "php": "^7.1",
         "phpspec/prophecy": "^1.7",
         "phpunit/phpunit": "^6.0||^7.0",
-        "phpstan/phpstan": "^0.10"
+        "phpstan/phpstan": "^0.10.3"
```

Comment:
I'd say that using `^0.10.3@dev` would be more appropriated for the testing, just to avoid issues with other deps.